docs/content/operator/validator-node/configs/snapshots.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Snapshots
+tags: [node-operation]
+teams:
+  - iotaledger/node
+description: Snapshots of the IOTA network enable full node operators a way to bootstrap a full node without having to execute all the transactions that occurred after genesis.
```

Comment:
```suggestion
description: Snapshots of the IOTA network enable validator node operators a way to bootstrap a full node without having to execute all the transactions that occurred after genesis.
```

Reply:
Fixed in ede15b84a110e29ed56b96fd6ceade6336c84587